### ISSUE TITLE
Nether Portals dont link 1:1

### ISSUE THREAD

**Issue by lipe123** [Multiverse/Multiverse-NetherPortals#80]
I looked at issue #68 but thats not my problem.

I have a amplified 1.7 world called "ampworld" and created a nether for it with /mc create amphell nether.

Then i linked them and went into a  nther portal, all fine a portal appears in the nether etc. I travel back via that exact same portal and it creates an entirely new portal about 200 blocks from the original one I went in from :(
The new portal that was created 2nd in the overworld always links back to the nether one.

TLDR: portals from the nether to overworld work fine. Portals from overworld to nether dont link back to themselves.
